### DISCUSSION TITLE
课程4：Colab 运行 vm["main"] 时崩溃

### DISCUSSION THREAD

**Discussion by KangShengYu** [mlc-ai/mlc-zh#139] [Q&A]
如图所示，重试了多次都没有成功，查看运行时日志也没有发现有用的信息。

<img width="941" alt="image" src="https://user-images.githubusercontent.com/80497369/187876909-436510d8-bae5-46c8-ae63-476c990068b0.png">
<img width="553" alt="image" src="https://user-images.githubusercontent.com/80497369/187876993-01da8fe5-bcc0-473b-b03b-7e6c9d305148.png">

**Comment by Hzfengsy** [comment]
刚刚测试了官方提供的notebook，在colab中可以运行，请检查是否脚本存在问题
